<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-UCS52DJ" failures="1" tests="6" name="Test" time="0.114" errors="0" timestamp="01 Sep 2018 17:59:59 GMT">
  <testcase classname="com.mycompany.tesNG.TestBase" name="testLogin" time="0.009"/>
  <testcase classname="com.mycompany.tesNG.TestBase" name="testAddCart" time="0.01"/>
  <testcase classname="com.mycompany.tesNG.Yahoo" name="testCompose2" time="0.018">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.mycompany.tesNG.Yahoo.testCompose2(Yahoo.java:15)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testCompose2 -->
  <testcase classname="com.mycompany.tesNG.Yahoo2" name="testCompose" time="0.002"/>
  <testcase classname="com.mycompany.tesNG.TestBase" name="testAddCart" time="0.026"/>
  <testcase classname="com.mycompany.tesNG.TestBase" name="testLogin" time="0.029"/>
</testsuite> <!-- Test -->
